src/kvirc/sparser/KviIrcServerParser_literalHandlers.cpp

```diff
@@ -954,9 +954,13 @@ void KviIrcServerParser::parseLiteralPrivmsg(KviIrcMessage * msg)
 		//			}
 		//		}
 
+		// "znc.in/self-message" capability: Handle a replayed message from ourselves to someone else.
+		bool bSelfMessage = IS_ME(msg, szNick);
+		QString szWindow = bSelfMessage ? szTarget : szNick;
```

Comment:
It's not about software, but about the spec. Even if you didn't see ZNC sending such messages, theorecitally it *can*.
And take your time for vacation, it's not urgent anyway :)